<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewClass</class>
 <widget class="QWidget" name="NewClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>855</width>
    <height>534</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Новый класс</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../build-IDE-Desktop_Qt_5_15_2_MinGW_64_bit-Debug/icon.png</normaloff>../build-IDE-Desktop_Qt_5_15_2_MinGW_64_bit-Debug/icon.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="classInformation">
     <property name="verticalSpacing">
      <number>40</number>
     </property>
     <property name="leftMargin">
      <number>25</number>
     </property>
     <property name="topMargin">
      <number>25</number>
     </property>
     <property name="rightMargin">
      <number>25</number>
     </property>
     <property name="bottomMargin">
      <number>25</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="labelNameClass">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Trebuchet MS</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Название класса</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="editNameClass">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Trebuchet MS</family>
         <pointsize>9</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelHeader">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Trebuchet MS</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Заголовочный</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="editHeader">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Trebuchet MS</family>
         <pointsize>9</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelSource">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Trebuchet MS</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Исходный</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="editSource">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Trebuchet MS</family>
         <pointsize>9</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonsBlock">
     <property name="leftMargin">
      <number>25</number>
     </property>
     <property name="rightMargin">
      <number>25</number>
     </property>
     <property name="bottomMargin">
      <number>25</number>
     </property>
     <item alignment="Qt::AlignLeft|Qt::AlignBottom">
      <widget class="QPushButton" name="cancelButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Trebuchet MS</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignRight|Qt::AlignBottom">
      <widget class="QPushButton" name="readyButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Trebuchet MS</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Готово</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
